rust/benchmarks/src/bin/tpch.rs

```diff
@@ -1636,7 +1636,7 @@ mod tests {
                 .file_extension(".out");
             let df = ctx.read_csv(&format!("{}/answers/q{}.out", path, n), options)?;
             let df = df.select(
-                &get_answer_schema(n)
```

Comment:
this is a pretty good example -- the `answer_schema` was created and immediately dropped after calling `df.select()`. Now the exprs don't need to be copied